src/js/components/sharedComponents/checkbox/AccordionCheckbox.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * CheckboxOnePrimary.jsx
```

Comment:
need to rename to match the component name

Reply:
renamed